Packs/remedy_SR/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### remedy_sr_beta
```

Comment:
What would it take to change the name of the integration?

Reply:
Not possible at the moment